conan/tools/cmake/cmakedeps2/cmakedeps.py

```diff
@@ -59,6 +59,14 @@ def _content(self):
             cmake_find_mode = cmake_find_mode.lower()
             if cmake_find_mode == FIND_MODE_NONE:
                 continue
+            if cmake_find_mode in (FIND_MODE_MODULE, FIND_MODE_BOTH):
+                ConanOutput(self._conanfile.ref).warning("CMakeConfigDeps does not support "
```

Comment:
This might create a bunch of deprecation warnings for recipes in ConanCenter that define the ``cmake_find_mode``, that we don't plan to change that ``cmake_find_mode`` anytime soon, I think? Probably for BOTH is not an issue at all? (I understand that the warning for ``MODULE`` only might make sense

Reply:
Still warns for BOTH?

Reply:
Missed to stash this oops, done now, good catch!